qml/DomainGraphLayout.qml

```diff
@@ -995,6 +996,27 @@ Item
                                                         Layout.rightMargin: spacing_icon_label_ + first_indentation_
                                                     }
                                                 }
+                                                Rectangle {
+                                                    id: participant_app_icon
+                                                    visible: modelData["app_id"] != "UNKNOWN_APP"
+                                                    anchors.right: parent.right
+                                                    anchors.verticalCenter: parent.verticalCenter
+                                                    width: participant_app_icon.visible ? icon_size_ + 2* spacing_icon_label_ : 0
+                                                    height: icon_size_
+                                                    color: "transparent"
+
+                                                    Image {
+                                                        smooth: true
+                                                        visible: parent.visible
+                                                        anchors.horizontalCenter: parent.horizontalCenter
+                                                        source: "/resources/images/app_icons/" + modelData["app_id"] + ".svg"
+                                                        readonly property int amlip_offset_: 5
+                                                        sourceSize.width: modelData["app_id"] == "AML_IP"
+                                                                ? parent.height + amlip_offset_ : parent.height
+                                                        sourceSize.height: modelData["app_id"] == "AML_IP"
+                                                                ? parent.height + amlip_offset_ : parent.height
```

Comment:
What is this for?

Reply:
Almost all icons has the same aspect ratio (more or less the same width than height) but AML_IP icon has wide more width than height. As long as all icons must be defined in a square size (100x100px for instance), this section makes the AML_IP icon to be a little more bigger just to make it look like the same size as the other icons

Reply:
If the image cannot be redrawn, add an explanatory comment similar to this one